solutions/swb-reference/package.json

```diff
@@ -17,7 +17,7 @@
     "build": "heft build --clean && rushx pkg-json-lint",
     "build:test": "heft test --clean && rushx pkg-json-lint && rushx make-badges",
     "cdk": "cdk",
-    "cdk-deploy": "rush build && rushx compile && cdk deploy --all --require-approval never --outputs-file ./src/config/${STAGE}.json",
+    "cdk-deploy": "rush build && rushx compile && cdk deploy --all --require-approval never --outputs-file ./src/config/byon.json",
```

Comment:
I think we want to keep this STAGE specific, right?

Reply:
yeah, sorry package.json gets merged automatically and I do this because the `STAGE` variable doesnt work in windows. reverting!